src/main/kotlin/uk/gov/dluhc/printapi/service/AedDataRetentionService.kt

```diff
@@ -50,4 +69,16 @@ class AedDataRetentionService(
             }
         }
     }
+
+    fun AnonymousElectorDocument.removeInitialRetentionPeriodData() {
```

Comment:
I didn't want to add this to `AnonymousElectorDocument` as it uses `addressRepository` and `deliveryRepository`, so have added it as an extension function (partly because I think it reads nicely above! 😄 ).